README.md

```diff
@@ -23,10 +23,19 @@ agent is NOT backwards-compatible with a config file from 0.1.4 and below.
 That being said, the difference is not huge, see below for an example on
 how to setup the new config file.
 
-```
-http://get.influxdb.org/telegraf/telegraf_0.1.5_amd64.deb
-http://get.influxdb.org/telegraf/telegraf-0.1.5-1.x86_64.rpm
-```
+As well, due to a breaking change to the InfluxDB integer line-protocol, there
+are some InfluxDB compatibility requirements:
+
+* InfluxDB <= 0.9.2 requires Telegraf <= 0.1.4
+* InfluxDB > 0.9.2 requires Telegraf > 0.1.4 (this includes any nightly builds)
```

Comment:
Cool, I'll update to reflect that! And as far as I know there is no reason to use Telegraf versions before 0.1.4